src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/CaseManagementLocationServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import junitparams.converters.Nullable;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.quality.Strictness;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.BaseLocation;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.CaseManagementLocation;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.Region;
+
+@RunWith(JUnitParamsRunner.class)
+public class CaseManagementLocationServiceTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule().strictness(Strictness.STRICT_STUBS);
+
+    private final CaseManagementLocationService service = new CaseManagementLocationService();
+
+    @Test
+    @Parameters({
+        "Birmingham, BIRMINGHAM",
+        "Glasgow, GLASGOW",
+        "Bradford, BRADFORD",
+        "Hatton Cross, HATTON_CROSS",
+        "Manchester, MANCHESTER",
+        "Newport, NEWPORT",
+        "Taylor House, TAYLOR_HOUSE",
+        "Newcastle, null"
```

Comment:
Can we also add Nottingham Justice Centre, Glasgow Tribunals Centre & Coventry Magistrates Courts too the parameters so that we capture all centres in test.

Reply:
Hi Ben, the above test is not testing hearing centres To location names mapping. It's testing StaffLocation names to Location name which is an intermediate step. 
What you are looking for is here https://github.com/hmcts/ia-case-api/blob/master/src/test/java/uk/gov/hmcts/reform/iacaseapi/utils/StaffLocationTest.java#L13 
Hope it makes sense.